src/Couchbase.Lite.Shared/Sync/ReplicatorOptionsDictionary.cs

```diff
@@ -81,6 +82,17 @@ internal sealed class ReplicatorOptionsDictionary : OptionsDictionary, IDisposab
 
         #region Properties
```

Comment:
Not sure what this is for... but it looks like it is probably doing the right thing...

Reply:
You mean #region?  It's just for code collapsing in IDEs

Reply:
No.. I meant the whole ReplicatorOptionsDictionary class, distinct from the ReplicatorConfiguration class.  I get it now, though.  There is an analog in Java... it is just written in C in the JNI code